### PR TITLE
Add the cultivated land eligibility rule

### PR DESCRIPTION
**Pull request by djbingham**
https://eaflood.atlassian.net/browse/FCEP-96

### CODE REVIEW

**Review by markysoft — DISMISSED** [review]
Looks good to me, passing the 'area minus 1' as the quantity was a little confusing, as it could have been defined at the top as a valid parcel area constant, but I'm happy for this to be merged without that change.

File: test/routes/paymentCalculation.integration.test.js [hunk 1]
```diff
@@ -288,5 +288,65 @@ describe('POST /parcels/{parcelRef/actions/{actionId}/payment-calculation', () =
...
+              rules: [
+                { id: 8, enabled: true }
+              ]
+            }
+          ]
+        )
+        await setUpServer()
+      })
+
+      test('parcel is eligible if parcel is arable land', async () => {
```
Comment (neilbmclaughlin) [inline test/routes/paymentCalculation.integration.test.js:312]: This test and all subsequent ones are testing both landCoverClass and area but the names don't reflect this.

**Review by neilbmclaughlin — CHANGES_REQUESTED** [review]
Some test name changes require but otherwise LGTM

File: test/routes/paymentCalculation.integration.test.js (hunk 1 above)
Comment (djbingham) [reply test/routes/paymentCalculation.integration.test.js:312]: I've only enabled the cultivated parcel rule, not any area rules, for these tests.

Comment (djbingham) [reply test/routes/paymentCalculation.integration.test.js:312]: I've removed the area calculation from those three tests in favour of passing an arbitrary value. That seems less confusing given that the area rules are disabled for those tests.

**Review by neilbmclaughlin — APPROVED** [review]
LGTM

**Pull request merged**